installer/install.sh

```diff
@@ -154,6 +192,25 @@ function check_oc() {
   fi
 }
 
+function check_oc_cluster_up() {
+  check_msg "Openshift cluster"
+  spinnerStart 'Running oc cluster up ...'
+  command oc cluster up &>/dev/null
```

Comment:
is ```oc cluster status``` a better option?

Reply:
Unfurtunilly, no. The only way to validate if the cluster is with all properly setup will works is by executing this command. However, note that it is executed in the ansible tasks too then we are just moving the place of the action and not adding more time/process to do it. 

The common issue that this PR solved is the bad experience of the end of this process face the following error.

TASK [oc-cluster-up : Cluster up] ```
*********************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "cmd": "oc cluster up --service-catalog=true --host-data-dir=/Users/cmacedo/mobile-core/ui/openshift-data --routing-suffix=192.168.1.9.nip.io --public-hostname=192.168.1.9 --version=v3.9.0 --image=openshift/origin", "delta": "0:00:00.276593", "end": "2018-07-12 09:04:46.472529", "msg": "non-zero return code", "rc": 1, "start": "2018-07-12 09:04:46.195936", "stderr": "error: FAIL\n   Error: did not detect an --insecure-registry argument on the Docker daemon\n   Solution:\n\n     Ensure that the Docker daemon is running with the following argument:\n     \t--insecure-registry 172.30.0.0/16\n\n     You can run this command with --create-machine to create a machine with the\n     right argument.", "stderr_lines": ["error: FAIL", "   Error: did not detect an --insecure-registry argument on the Docker daemon", "   Solution:", "", "     Ensure that the Docker daemon is running with the following argument:", "     \t--insecure-registry 172.30.0.0/16", "", "     You can run this command with --create-machine to create a machine with the", "     right argument."], "stdout": "", "stdout_lines": []}
	to retry, use: --limit @/Users/cmacedo/mobile-core/installer/playbook.retry
```

What is better with this PR?

* If the cluster cannot be up and will face some error it will be checked before many tasks be executed
* Here the error/reason is really clear
* Also, we could add the link with the info about how to solve the common issues.

PS.: The ansible tasks output should appear just when the script is executed in debug mode or --v, -vv.